### PR TITLE
PGOV-955 Update to D10 without jQuery.once

### PR DESCRIPTION
**Pull request by kxwang**

### CODE REVIEW

**Comment by kxwang** [comment]
@RichardDavies Please hold off on review. I just realized I need to merge with Master first.

**Comment by kxwang** [comment]
OK, resolved the merging conflicts.

**Comment by RichardDavies** [comment]
@kxwang 	FYI There's a merge conflict in composer.lock

File: composer.json [hunk 1]
```diff
@@ -175,10 +176,9 @@
     },
     "require-dev": {
         "drupal/coder": "^8.2",
-        "drupal/core-dev": "9.5.9",
+        "drupal/core-dev": "^10",
```
Comment (RichardDavies) [inline composer.json:179]: This can be removed now (or create a followup story)

**Review by RichardDavies — APPROVED** [review]
There are too many config changes to inspect them all, so I looked at the critical ones and spot checked a bunch of others. Found one minor dev dependency that could be removed either now or later from composer.json. But overall looks fine so approved.

File: composer.json (hunk 1 above)
Comment (kxwang) [reply composer.json]: Good call! I've removed "core-dev"

**Comment by kxwang** [comment]
Resolved the conflict on composer.lock

**Review by RichardDavies — APPROVED** [review]

**Pull request merged**